### PR TITLE
UX study updates

### PR DESCRIPTION
**Pull request by mmacy**
Integrates changes from learnings gained from internal pilot and external session 1.

### CODE REVIEW

**Comment by mmacy** [comment]
@johanste @binderjoe Integrating some of the changes discussed during yesterday's pilot session with Anna and Laurent.

**Comment by mmacy** [comment]
@johanste Made a few more updates, this is ready for review.

File: README.md [hunk 1]
```diff
@@ -167,20 +140,32 @@ for i in range(1, 10):
...
+database = client.get_database(database_name)
+container = database.get_container(container_name)
+
+# Enumerate the returned items
+import json
+for item in container.query_items(query='SELECT * FROM mycontainer r WHERE r.id="something"'):
+    print(json.dumps(item, indent=True))
 ```
 
+> NOTE: Although you can specify *any* value for the container name in the `FROM` clause, we recommend you use the container name for consistency.
```
Comment (johanste) [inline README.md:159]: One thing that bit me was when I used a reserved name for the alias (order in my case). This made the SQL query syntax invalid.

**Review by johanste — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mmacy) [reply README.md:159]: Ah yes, good point! Will re-word on next pass.

**Pull request merged**